go.mod

```diff
@@ -155,5 +155,6 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/cosmos/cosmos-sdk => github.com/tharsis/cosmos-sdk v0.45.2-0.20220306021140-2de5a020cedb
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/tharsis/ethermint => github.com/loredanacirstea/ethermint v0.6.1-0.20220404144557-bf7358bae637
```

Comment:
do we need to revert this?

Reply:
Exactly. So I changed the `PostTxProcessing` hook to include the entire message. Might be of use for others too, to have access to both message and receipt.